source/agg-pipeline-builder/open-saved-pipeline.txt

```diff
@@ -12,4 +12,56 @@ Open a Saved Pipeline
    :depth: 1
    :class: singlecol
 
-TODO:
+Instead of creating a new pipeline from the beginning, you can load and edit 
+saved aggregation pipelines.
+
+Before You Begin 
+----------------
+
+If you don't already have a saved pipeline, you must first: 
+
+- :ref:`Create an aggregation pipeline <create-agg-pipeline>`.
+
+- :ref:`Save your aggregation pipeline <save-agg-pipeline>`.
+
+Procedure
+---------
+
+.. procedure:: 
+   :style: normal
+
+   .. step:: Click the folder icon.
+
+      In the top-left corner of the pipeline builder, click the folder icon to 
+      open the drop-down menu of saved pipelines.
+      
+      .. figure:: /images/compass/saved-pipeline-icon.png
+         :alt: Saved Pipeline Icon
+         
+   .. step:: Select an aggregation pipeline.
+      
+      In the drop-down menu, hover over the pipeline you want to open and 
+      click :guilabel:`Open`.  
+      
+      .. figure:: /images/compass/saved-pipelines.png
+         :alt: Saved Pipelines List 
+         
+   .. step:: Confirm your selection. 
+      
+      In the pop-up modal, click :guilabel:`Open Pipeline`.
+
+.. note::
+
+   Starting in |compass| 1.31, you can view your saved queries
+   and aggregation pipelines on the :guilabel:`My Queries` view 
```

Comment:
Should this be 
in the My Queries view
or 
on the My Queries view [tab | screen | window]

Reply:
will update to tab.

Reply:
just confirmed with Julia that it wasn't an explicit product decision — will be changing these to "tab" across the relevant docs